pkg/cloud/volume_limits.go

```diff
@@ -551,3 +556,21 @@ var gpuInstanceGpus = map[string]int{
 	"p4de.24xlarge": 8,
 	"p5.48xlarge":   8,
 }
+
+// Note this table is not a reflection of how many accelerators an instance has but of how many slots their combined acclerators take up
+// VT instance type acclerators take two slots each with the exception of the vt1.24xlarge which takes 0 slots for its acclerators
```

Comment:
Latest rev adds a typo here: "acclerators"

Also, we should squash before merge. Otherwise lgtm.